ckanext/csrf_filter/anti_csrf.py

```diff
@@ -90,6 +96,13 @@ def configure(config):
     token_expiry_age = 60 * config.get('ckanext.csrf_filter.token_expiry_minutes', 30)
     token_renewal_age = 60 * config.get('ckanext.csrf_filter.token_renewal_minutes', 10)
 
+    exempt_rules = config.get('ckanext.csrf_filter.exempt_rules', None)
+    if exempt_rules is not None:
```

Comment:
This could be shortened to something like:

    if exempt_rules:
        exempt_rules = [re.compile(rule) for rule in exempt_rules.split()]

And it should probably be in a separate pull request.